### PR TITLE
React capstone project - Covid 19 CDR

### PR DESCRIPTION
**Pull request by atok624**
In this pull request, I completed the following tasks:
- Ensured that the project is a single-page application (SPA) built with React and Redux.
- Ensured that the data I retrieved from the API was stored in the Redux store.
- I filtered the data that I retrieve from the API using a Filter stateless component.
- Ensured every page (the main page and the pages for each item) had a unique route within the SPA.
- Ensured that the app was deployed and was accessible online using [Netlify](https://www.netlify.com/)
- Added a link to the online version of my app to your README file.

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+  if (state === 'Success') {
+    content = (
+      <div className="second-row">
+        {data.rawData
+          .slice(0, 700)
+          .filter((x) => x.Country_Region.toLowerCase().includes(searchData.toLowerCase()))
+          .map((item, index) => (
+            <Section
+              key={uuidv4()}
```
Comment (ghost) [inline src/components/Home.js]: - Kindly use React router dom to handle client-side routing.   React Router DOM allows you to define routes declaratively as components, making it easier to manage and reuse routes throughout your application. It also allows you to pass parameters in the URL, which makes it easy to build dynamic and interactive applications.

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    content = (
+      <div className="details-row">
+        <Component
+          date={data.rawData[indexOfData].Last_Update}
+          country={data.rawData[indexOfData].Combined_Key}
+          incident={parseFloat(data.rawData[indexOfData].Incident_Rate).toFixed(
+            4,
+          )}
+          cases={data.rawData[indexOfData].Confirmed}
+          deaths={data.rawData[indexOfData].Deaths}
```
Comment (ghost) [inline src/components/Home.js]: - [ ] Generally, you can improve the overall design of your app by:
  - Checking the color contrast between the names of the countries and the background. Low-contrast text can be difficult to read, which can make it challenging for users with color blindness, low vision, or other visual impairments to access and use the content on your website.
  - Adjusting the design of your navigation bar to match the original design in terms of spacing of elements
      
image

### Checkout the original design

image

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @atok624 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good Documentation👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by atok624** [comment]
Thanks @frank1738 for the review

**Review by jerryowusu — APPROVED** [review]
Hi @atok624 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
:heavy_check_mark: Neat and well organized code 
:heavy_check_mark: Nice design 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by atok624** [comment]
Thanks so much, @jerryowusu 
I am so grateful for this review

**Pull request merged**